build.gradle

```diff
@@ -0,0 +1,168 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.6.21'
```

Comment:
If we use kotlin, please, configure the target to match the Java target (java 11)